.changeset/tiny-spoons-repeat.md

```diff
@@ -0,0 +1,15 @@
+---
+'@backstage/backend-defaults': patch
+'@backstage/plugin-catalog-backend': minor
+---
+
+**BREAKING**: The `UrlReaderProccessor` accepts a new config flag `catalog.useUrlReadersSearch` to always call the `search` method of `UrlReaders`.
```

Comment:
this doesn't look like breaking at the moment, right? 🤔

Reply:
Yes you're right, I removed it!